components/infobox/wikis/stormgate/infobox_unit_custom.lua

```diff
@@ -64,57 +64,74 @@ function CustomInjector:parse(id, widgets)
 
 	if id == 'type' then
 		return {
-			Cell{name = 'Type', content = caller:_displayCommaSeparatedString(args.type)},
+			Cell{name = 'Type', content = {caller:_displayCommaSeparatedString(args.type)}},
 		}
 	elseif id == 'builtfrom' then
 		return {
-			Cell{name = 'Built From', content = caller:_displayCommaSeparatedString(args.built)},
+			Cell{name = 'Built From', content = {caller:_displayCommaSeparatedString(args.built)}},
 		}
 	elseif id == 'requirements' then
 		return {
-			Cell{name = 'Tech. Requirement', content = caller:_displayCommaSeparatedString(args.tech_requirement)},
-			Cell{name = 'Building Requirement', content = caller:_displayCommaSeparatedString(args.building_requirement)},
+			Cell{name = 'Tech. Requirement', content = {caller:_displayCommaSeparatedString(args.tech_requirement)}},
+			Cell{name = 'Building Requirement', content = {caller:_displayCommaSeparatedString(args.building_requirement)}},
 		}
 	elseif id == 'cost' then
 		return {
-			Cell{name = 'Cost', content = {CostDisplay.run{
-				faction = caller.faction,
-				luminite = args.luminite,
-				luminiteTotal = args.totalluminite,
-				luminiteForced = true,
-				therium = args.therium,
-				theriumTotal = args.totaltherium,
-				theriumForced = true,
-				supply = args.supply,
-				supplyTotal = args.totalsupply,
-				supplyForced = true,
-				animus = args.animus,
-				animusTotal = args.totalanimus,
-			}}},
-			Cell{name = 'Build Time', content = {args.buildtime and (args.buildtime .. 's') or nil}},
-			Cell{name = 'Recharge Time', content = {args.charge_time and (args.charge_time .. 's') or nil}},
+			Cell{
+				name = 'Cost',
+				content = {
+					args.informationType ~= 'Hero' and CostDisplay.run{
+						faction = caller.faction,
+						luminite = args.luminite,
+						luminiteTotal = args.totalluminite,
+						luminiteForced = true,
+						therium = args.therium,
+						theriumTotal = args.totaltherium,
+						theriumForced = true,
+						supply = args.supply,
+						supplyTotal = args.totalsupply,
+						supplyForced = true,
+						animus = args.animus,
+						animusTotal = args.totalanimus,
+					} or nil
+				}
+			},
+			Cell{
```

Comment:
code style of this cell seems prett bad
also as per discussion on discord maybe change to moving buildtime into costs cell

Reply:
Moved builtime to costs, changed order of costs in extension by showing supply before buildtiume and changed coide sytle (like in other files).